package/src/ssr/hooks.ts

```diff
@@ -21,6 +24,13 @@ export const useSuspenseQuery = wrap(orig_useSuspenseQuery, [
   "data",
   "networkStatus",
 ]);
+export const useReadQuery = wrap(orig_useReadQuery, ["data", "networkStatus"]);
+// @ts-ignore
+export const useBackgroundQuery: typeof orig_useBackgroundQuery = (...args) => {
+  useRehydrationContext();
+  // @ts-ignore
+  return orig_useBackgroundQuery(...args);
+};
```

Comment:
let's get rid of those `@ts-ignore` annotations so eslint shuts up :)
```suggestion
export const useBackgroundQuery: typeof orig_useBackgroundQuery = (...args: [any, any]) => {
  useRehydrationContext();
  return orig_useBackgroundQuery(...args);
};
```

Reply:
Resolved in [867d581](https://github.com/apollographql/apollo-client-nextjs/pull/38/commits/867d5812951b586c6c8cc23fba07e82e0dd129bf)